.github/CODEOWNERS

```diff
@@ -6,7 +6,7 @@
 ################
 
 # Git Hub integration and bot rules
-/.github/                @AlexGhiondea @jsquire @jhendrixMSFT @rickwinter
+/.github/                    @jsquire @jhendrixMSFT @rickwinter
```

Comment:
Is this errant whitespace or does it now line up with other things?

Reply:
Lines up with the rest of the file